RecoMuon/MuonSeedGenerator/plugins/MuonSeedGenerator.cc

```diff
@@ -107,8 +107,18 @@ void MuonSeedGenerator::produce(edm::Event& event, const edm::EventSetup& eSetup
 void MuonSeedGenerator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.setAllowAnything();
```

Comment:
Thank you for having added the missing parameter descriptions.

At this point, you can even complete the job and also add the two PSet's `ptSeedParameterization` and `dphiScale,`. They could even be empty PSet's here in the fillDescriptions method, and they will get then filled with the actual values only in the explicit python configurations.  E.g.:
```C++
   {
      edm::ParameterSetDescription psd0;
      psd0.setUnknown();
      desc.add<edm::ParameterSetDescription>("ptSeedParameterization", psd0);
   }
   {
      edm::ParameterSetDescription psd0;
      psd0.setUnknown();
      desc.add<edm::ParameterSetDescription>("dphiScale", psd0);
   }
```

If you do so, you can then remove the line:
```C++
  desc.setAllowAnything();
```
If everything is correctly updated, this should work